src/HttpClient/HttpClientFactory.php

```diff
@@ -103,7 +103,7 @@ public function create(Options $options): HttpAsyncClientInterface
         if (null === $httpClient) {
             if (class_exists(SymfonyHttplugClient::class)) {
                 $symfonyConfig = [
-                    'max_duration' => self::DEFAULT_HTTP_TIMEOUT,
+                    'max_duration' => $options->getHttpTimeout(),
                 ];
```

Comment:
That's strange... The Symfony client has no connect timeout setting? Does this mean that it conflates the two values? Should we sum the two values from the config here? Should we leave it like that?

Reply:
Agreed. I had a look through the Symfony documentation: https://symfony.com/doc/current/http_client.html#dealing-with-network-timeouts

This mentions both the `max_duration` and `timeout`, from what I can gather, `max_duration` is a governing variable, so summing, while it feels a bit weird is probably the right way to go. That's done in `e5f9627`.